ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1192,6 +1193,15 @@ ip_icmp_redirect: < true | false >
 ipv6_icmp_redirect: < true | false >
 ```
 
+### LACP
+
+```yaml
+lacp:
+  port_id_min: < min_port >
+  port_id_max: < max_port >
```

Comment:
We should keep structure similar to EOS configuration:
```POD1_LEAF01A(config)#lacp ?
  port-id          LACP port-id range configuration
  rate-limit       Set LACPDU Rate Limit Options
  system-priority  Set local system LACP priority

POD1_LEAF01A(config)#lacp port-id ?
  range  LACP port-id range configuration

POD1_LEAF01A(config)#lacp port-id range ?
  <1-65535>  LACP port-ID range```

I suggest we ident port-id variables into port_id and then into range to allow for future extensibility if required.

Also, do we want to make rate-limit configurable?

Reply:
rate-limit doesn't make any difference even if I configure because it is set to default only. I don't know the reason. But I will include it.